ndn_python_repo/storage/storage_base.py

```diff
@@ -118,7 +120,7 @@ def get_data_packet(self, name: NonStrictName, can_be_prefix: bool=False,
                     while True:
                         data, expire_time_ms = next(it)
                         if not must_be_fresh or expire_time_ms > self._time_ms():
-                            self.logger.info('get from cache')
+                            self.logger.info(f'get from cache')
```

Comment:
nit: Some linter will complain if you put an `f` prefix to a string literal without `{}`.